File: OneDrive/Documentos/Store_Herencia/Store_Herencia/src/store/herencia/Cleaning.java

package store.herencia;

public class Cleaning extends Product {
    private String brand;
    public Cleaning(String name, double price, String importDate, int stock, String brand) {
        super(name, price, importDate, stock);
        this.brand = brand;
    }

    @Override
    public String toString() {
        return super.getData() + String.format(", brand: %s", brand);
    }
}

File: PycharmProjects/Store_Herencia/py/Cleaning.py
from Product import Product

class Cleaning(Product):
    def __init__(self, name, price, import_date, stock, brand):
        super().__init__(name, price, import_date, stock)
        self.brand = brand

    def __str__(self):
        return f"{super().get_data()}, brand: {self.brand}"
